dd-sdk-android/src/test/kotlin/com/datadog/android/utils/DatadogExt.kt

```diff
@@ -16,7 +17,31 @@ import com.nhaarman.mockitokotlin2.whenever
 import java.io.File
 
 /**
- * Mocks a context with the minimal behavior to initialize the Datadog library.
+ * Mocks an AppContext with the minimal behavior to initialize the Datadog library.
+ */
+fun mockAppContext(
```

Comment:
Again, instead of rewriting everything twice, maybe you can just modify the `mockContext` so that it returns an Application ? Or at least reuse some of the code, cause now whenever we need to change the mocked context for initialization, we need to change it twice.